CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 - [#5901](https://github.com/influxdata/telegraf/pull/5901): Set user agent in stackdriver output.
 - [#5885](https://github.com/influxdata/telegraf/pull/5885): Extend metrics collected from Nvidia GPUs.
 - [#5547](https://github.com/influxdata/telegraf/pull/5547): Add file rotation support to the file output.
+- [#5945](https://github.com/influxdata/telegraf/pull/5945): Upgrade from aws-sdk-go 1.15.54 to 1.19.41.
```

Comment:
Can you remove this, we would only add it to the changelog if it addressed a particular bug or feature.  Also it's just a pain to avoid conflicts in this file so we usually update it post merge.

Reply:
Dropped this.